### ISSUE TITLE
There's a duplicate config for the consul ssl, line 25-28 & 43-46.

### ISSUE THREAD

**Issue by lanefu** [ansible-community/ansible-nomad#146]
There's a duplicate config for the consul ssl, line 25-28 & 43-46.

_Originally posted by @kedai in https://github.com/ansible-community/ansible-nomad/pull/129#issuecomment-1004557766_

**Comment by ppacher** [comment]
This acutally prevents my nomad cluster from starting because the nodes complain with the following error:

```
==> Error loading configuration from /etc/nomad.d: Error loading /etc/nomad.d/base.hcl: consul unexpected keys ca_file, cert_file, key_file, ssl
```

**Comment by lanefu** [comment]
Resolved #148
